Adjust VM heap growth factor

hyjorc1 opened the pull request:
#### 34d0653dff22775b387c17d94eb63277fa7ac3c0
<pre>
Adjust VM heap growth factor
<a href="https://bugs.webkit.org/show_bug.cgi?id=242347">https://bugs.webkit.org/show_bug.cgi?id=242347</a>

Reviewed by Yusuke Suzuki.

We adjust VM heap growth factor to gain 0.5% RAMification progression.

* Source/JavaScriptCore/runtime/OptionsList.h:

Canonical link: <a href="https://commits.webkit.org/252169@main">https://commits.webkit.org/252169@main</a>
</pre>

Constellation reviewed (approved):
r=me if watch-cli.js is neutral.
Let's measure non-JIT watch-cli.js JetStream2 score to ensure that we do not introduce significant regression.
And let's paste the result data to commit message.

hyjorc1 committed:
Adjust VM heap growth factor
https://bugs.webkit.org/show_bug.cgi?id=242347

Reviewed by Yusuke Suzuki.

We adjust VM heap growth factor to gain 0.5% RAMification progression.

* Source/JavaScriptCore/runtime/OptionsList.h:

Canonical link: https://commits.webkit.org/252169@main
